### PR TITLE
Log more information when task ID/ namespace ID collision occurs

### PR DESCRIPTION
**Pull request by laniehei**
## What changed?
Added logging during task ID/ namespace ID collision 

## Why?
debugging purposes 

## How did you test it?
N/A

## Potential risks
small/no risk 

## Is hotfix candidate?
no

### CODE REVIEW

File: common/namespace/replication_task_executor.go [hunk 1]
```diff
@@ -130,6 +131,7 @@ func (h *namespaceReplicationTaskExecutorImpl) shouldProcessTask(ctx context.Con
 	switch err.(type) {
 	case nil:
 		if resp.Namespace.Info.Id != task.GetId() {
+			h.logger.Error("namespace replication encountered name / UUID collision during determination on whether to process task", tag.NewStringTag("namespaceID", task.Info.GetName()), tag.NewErrorTag(err))
```
Comment (yux0) [inline common/namespace/replication_task_executor.go:134]: nit: tag.tag.WorkflowNamespaceID or tag.WorkflowNamespace

Comment (yux0) [reply common/namespace/replication_task_executor.go:134]: nit: the error is nil at this line, no need to log error

**Review by yux0 — APPROVED** [review]

File: common/namespace/replication_task_executor.go (hunk 1 above)
Comment (laniehei) [reply common/namespace/replication_task_executor.go:134]: thanks! fixed both

File: common/namespace/replication_task_executor.go [hunk 2]
```diff
@@ -210,6 +214,7 @@ func (h *namespaceReplicationTaskExecutorImpl) handleNamespaceCreationReplicatio
 		switch getErr.(type) {
 		case nil:
 			if resp.Namespace.Info.Name != task.Info.GetName() {
+				h.logger.Error("namespace replication encountered name / UUID collision during NamespaceCreationReplicationTask (second GetNamespace call)", tag.WorkflowNamespaceID(task.Info.GetName()), tag.NewErrorTag(err), tag.NewErrorTag(getErr))
```
Comment (yux0) [inline common/namespace/replication_task_executor.go:217]:
```suggestion
				h.logger.Error("namespace replication encountered name collision during NamespaceCreationReplicationTask", tag.WorkflowNamespaceIDs(map[string]struct{}{resp.Namespace.Info.Name: {}, task.Info.GetName(): {}}), tag.NewErrorTag(err))
```

**Pull request merged**